src/main/java/bio/terra/common/logging/GoogleJsonLayout.java

```diff
@@ -57,18 +60,23 @@
  */
 class GoogleJsonLayout extends JsonLayoutBase<ILoggingEvent> {
 
-  private Tracer tracer = Tracing.getTracer();
-
   // A reference to the current Spring app context, on order to pull out the spring.application.name
   // and spring.application.version variable for inclusion in JSON output.
   private ConfigurableApplicationContext applicationContext;
```

Comment:
It's nice that you're able to get JSON into stackdriver without using using their logging client everywhere.